.travis.yml

```diff
@@ -250,8 +250,7 @@ before_script:
   # Print debug system information
   - cat /proc/sys/kernel/core_pattern
   - cat /etc/default/apport || true
-  - service --status-all || true
-  - initctl list || true
+  - systemctl list-units --type=service --state=running
```

Comment:
Not sure if this was intentional, but you removed the `true` which will bail if `systemctl(1)` bails.

Reply:
Thanks for spotting this, I'll change it now.